nautobot/project-static/css/base.css

```diff
@@ -42,7 +42,7 @@ body {
     transition-duration: var(--navbar-transition-duration);
     transition-timing-function: ease-in-out;
     transition-property: margin-left, width;
-    overflow: visible;
+    overflow: hidden;
```

Comment:
This change hides the extra rows when the footer wraps to multiple lines on mobile devices or very narrow windows - I don't think it's desirable.